src/snowflake/snowpark/_internal/analyzer/snowflake_plan_node.py

```diff
@@ -169,7 +175,7 @@ def __init__(
         table_type: str = "",
         clustering_exprs: Optional[Iterable[Expression]] = None,
         comment: Optional[str] = None,
-        is_generated: bool = False,
+        creation_source: TableCreationSource = TableCreationSource.EXPLICIT_USER_COMMAND,
```

Comment:
I prefer not having a default value so developer will be asked to set it explicitly